ada/testsuite/tests/properties/is_dispatching_call/pkg.adb

```diff
@@ -0,0 +1,29 @@
+package body Pkg is
+   procedure Test is
+      type F_A_Type is access procedure (Self : T);
+
+      X : T'Class;
+      Y : T;
+      I : Int;
+      F_A : F_A_Type;
+   begin
+      -- normal dispatching proc
```

Comment:
Following live discussions:

1. Please check that your code compiles with GNAT
2. Please add the correct example of dispatching  on return type
3. Personal taste, but if possible, I prefer when test cases fit in one file, makes them much easier to browse.